blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -1,123 +1,252 @@
 import React from "react";
-import PropTypes from "@arc-fusion/prop-types";
-import { useContent } from "fusion:content";
 
+import { useContent, useEditableContent } from "fusion:content";
 import { useFusionContext } from "fusion:context";
-import { ExtraLargePromoPresentation } from "@wpmedia/shared-styles";
-import { LazyLoad, isServerSide, videoPlayerCustomFields } from "@wpmedia/engine-theme-sdk";
-import { imageRatioCustomField } from "@wpmedia/resizer-image-block";
+import getTranslatedPhrases from "fusion:intl";
+import getProperties from "fusion:properties";
+
+import PropTypes from "@arc-fusion/prop-types";
+import {
+	Attribution,
+	Conditional,
+	Date as DateComponent,
+	formatAuthors,
+	getVideoFromANS,
+	Heading,
+	HeadingSection,
+	Image,
+	Join,
+	isServerSide,
+	Link,
+	MediaItem,
+	Overline,
+	Paragraph,
+	Separator,
+	Stack,
+	Video,
+} from "@wpmedia/arc-themes-components";
+
+import { LazyLoad, localizeDateTime } from "@wpmedia/engine-theme-sdk";
+import { extractImageFromStory } from "@wpmedia/resizer-image-block";
```

Comment:
Can we use the Arc Themes Component - `getImageFromANS` ?

Reply:
oh yeah, I missed this.

Reply:
done